WordPress/Classes/Models/Blog.m

```diff
@@ -259,6 +259,18 @@ - (NSArray *)sortedPostFormatNames
     }];
 }
 
+- (NSArray *)sortedConnections
+{
+    NSSortDescriptor *sortServiceDescriptor = [[NSSortDescriptor alloc] initWithKey:@"service"
+                                                                          ascending:YES
+                                                                           selector:@selector(caseInsensitiveCompare:)];
+    NSSortDescriptor *sortExternalNameDescriptor = [[NSSortDescriptor alloc] initWithKey:@"externalName"
+                                                                               ascending:YES
+                                                                                selector:@selector(caseInsensitiveCompare:)];
```

Comment:
We should use `localizedCaseInsensitiveCompare:` only for the service name, right? We don't want to modify the external account name which is what the user is used to expect as a visible name. Even though the chances of an external name being in the localized strings file there's no point in using `localizedCaseInsensitiveCompare:` for non localizable strings.